test/phpunit/SecurityTest.php

```diff
@@ -91,7 +91,7 @@ public function __construct()
      *
      * @return void
      */
-    public static function setUpBeforeClass()
+    public static function setUpBeforeClass():void
```

Comment:
travis check in develop branch are also done with php5.6

Reply:
my bad I forgot to remove them